pyof/v0x04/asynchronous/error_msg.py

```diff
@@ -435,7 +435,7 @@ def __init__(self, xid=None, error_type=None, code=None, data=b''):
         super().__init__(xid)
         self.error_type = error_type
         self.code = code
-        self.data = data
+        self.data = b'' if data is None else data
```

Comment:
The cleaner way is to set `b''` directly in the `__init__` method - same behavior, better readability.